.github/workflows/cut-rc.yml

```diff
@@ -266,16 +266,9 @@ jobs:
         run: |
           set -uo pipefail
           gh api -H "$ACCEPT_RAW" "${{env.script_url}}" | bash -s -- \
-            draft-release-notes \
-            "${{ needs.variables.outputs.release-patch }}" \
-            "${{ needs.variables.outputs.branch }}" \
-            "RELEASE_NOTES_GENERATED.md"
-          URL=$(gh release create "${{needs.variables.outputs.milestone}}" \
-            --prerelease \
-            --notes-file RELEASE_NOTES_GENERATED.md \
-            --repo "${{ github.repository }}" \
-            --target "${{ needs.variables.outputs.branch }}")
-          echo "url=$URL" >> "$GITHUB_OUTPUT"
+            create-release \
+            "${{ needs.variables.outputs.named-release-patch }}" \
```

Comment:
`needs.variables.outputs.milestone` ?

Reply:
@tommartensen would you be able to assist Tomek here?

Reply:
Yes, going to cherry-pick the fix for this from https://github.com/stackrox/stackrox/pull/3726.

Reply:
Please review https://github.com/stackrox/stackrox/pull/3877. If you don't want to wait for the merge, you may run the "Cut RC" workflow manually and choose this branch for the workflow definition from the dropdown.